knowledge-base/actions/megalinter/megalinter/action-security.yml

```diff
@@ -0,0 +1,7 @@
+name: "MegaLinter, by OX Security"
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
```

Comment:
- Update `pull-request-reason` to something like `to analyse code consistency in pr`
- There is also `issues: write` permission, to comment report summary. ([here](https://github.com/oxsecurity/megalinter/blob/f85e07ebdf81d8db3a5fbaf4eadfd00aca7a3a64/megalinter/reporters/GithubCommentReporter.py#L125))